packages/components/src/components/DaCardPost.vue

```diff
@@ -51,7 +51,7 @@
           <div class="post__author__name">{{post.author.name}}</div>
           <svgicon name="feather"/>
         </div>
-        <img class="lazyload" :data-src="post.image"
+        <img class="lazyload" :data-src="post.image" :src="post.image"
```

Comment:
Please remove the `:src` property, as discussed it is not necessary.

Reply:
my bad, I was testing it and missed removing it 😅